test/TestApps/ToastNotificationsTestApp/ToastNotificationsTestApp.cpp

```diff
@@ -186,6 +188,83 @@ bool VerifyFailedToastAssetsWithNullIconPath_Unpackaged()
     return false;
 }
 
+bool VerifyToastProperties()
+{
+    winrt::hstring xmlPayload{ L"<toast>intrepidToast</toast>" };
+
+    winrt::XmlDocument xmlDocument{};
+    xmlDocument.LoadXml(xmlPayload);
+    
+    winrt::ToastNotification toast(xmlDocument);
+
+    if (toast.Payload() != xmlDocument)
+    {
+        return false;
+    }
+
+    winrt::hstring tag{ L"tag" };
+    toast.Tag(tag);
+
+    if (toast.Tag() != tag)
+    {
+        return false;
+    }
+
+    winrt::hstring group{ L"group" };
+    toast.Group(group);
+
+    if (toast.Group() != group)
+    {
+        return false;
+    }
+
+    /*
+    * TODO: Uncomment once ToastProgressData has been implemented
+    winrt::ToastProgressData progressData{};
+    progressData.Status(L"SomeStatus");
+    progressData.Title(L"SomeTitle");
+    progressData.Value(0.14);
+    progressData.ValueStringOverride(L"14%");
+
+    toast.ProgressData(progressData);
+
+    auto progressDataFromToast = toast.ProgressData();
+    if (progressDataFromToast != progressData)
+    {
+        return false;
+    }
+    */
+
+    winrt::DateTime expirationTime = winrt::clock::now();
+    expirationTime += winrt::TimeSpan(std::chrono::seconds(10));
+
+    toast.ExpirationTime(expirationTime);
+    if (toast.ExpirationTime() != expirationTime)
+    {
+        return false;
+    }
+
+    toast.Priority(winrt::ToastPriority::High);
+    if (toast.Priority() != winrt::ToastPriority::High)
```

Comment:
Should we verify that the value is set the default before verifying that the value matches the value we just set? 

Same questions to most of the tests in test method.